Makefile

```diff
@@ -161,6 +165,12 @@ antctl-release:
 	@echo "==> Running unit tests <=="
 	$(GO) test -race -coverprofile=.coverage/coverage-unit.txt -covermode=atomic -cover github.com/vmware-tanzu/antrea/cmd/... github.com/vmware-tanzu/antrea/pkg/...
 
+.PHONY: .windows-test-unit
+.windows-test-unit:
+	@echo
+	@echo "==> Running unit tests <=="
+	$(GO) test github.com/vmware-tanzu/antrea/cmd/... github.com/vmware-tanzu/antrea/pkg/...
```

Comment:
Hi @jianjuns,
Is it intentional not to use -race flag for windows-test-unit target unlike linux-test-unit?
Seems like TestLargeScaleEndpointQueryManyPolicies cannot succeed consistently.. there are failures in different PRs.

Reply:
I missed it mistakenly. Will create a PR to add the flag.
But are you saying TestLargeScaleEndpointQueryManyPolicies is failing due the flag is removed?

Reply:
@tnqn : is it intended to bypass TestLargeScaleEndpointQueryManyPolicies for a PR? I know it might not work well with -race.

Reply:
With race flag, we [bypass](https://github.com/vmware-tanzu/antrea/blob/master/pkg/controller/networkpolicy/endpoint_querier_perf_test.go#L1) the tests in  endpoint_querier_perf_test.go.

Without the flag, I was able to run successfully on my local machine. However, the test is not consistently passing on PRs (with GitHub runners).. not sure why this is happening though. This test is added as part of #1021

Reply:
@jianjuns Yes, I bypass some large scale tests because they are only used to count execution time while `-race` slows it down greatly so meaningless to run them in CI. I usually run them separately to check the efficiency of some functions. I think `TestLargeScaleEndpointQueryManyPolicies` is similar and should be skipped.